libfwupd/meson.build

```diff
@@ -135,6 +135,7 @@ if get_option('introspection')
     includes : [
       'Gio-2.0',
       'GObject-2.0',
+#      'Json-1.0',
```

Comment:
if not needed, maybe just drop all together?

Reply:
We do need it, but including it breaks the vapi build further down. It's on the list to debug, although it looks like a vala issue really.

Reply:
Fixed.